src/states/common/relocated.rs

```diff
@@ -49,21 +50,9 @@ pub trait Relocated: Bootstrapped {
             self.peer_mgr_mut().set_connecting(their_pub_id);
         }
 
-        let our_conn_info = self.our_connection_info()?;
-        let shared_secret = self
-            .full_id()
-            .encrypting_private_key()
-            .shared_secret(&their_pub_id.encrypting_public_key());
-        let encrypted_conn_info = shared_secret.encrypt(&our_conn_info).map_err(|err| {
-            debug!(
-                "{} - Failed to serialise our connection info for {:?}: {:?}.",
-                self, their_pub_id, err
-            );
-            err
-        })?;
-
+        let conn_info = serialise(&self.our_connection_info()?)?;
         let content = MessageContent::ConnectionRequest {
-            encrypted_conn_info,
+            encrypted_conn_info: conn_info,
```

Comment:
Is the connection info no longer encrypted, then?

Reply:
No, it is open to everyone. It is just how it all works now, each node can connect to any other node direct. quic-p2p checks for 2 way connections to prevent network imbalance of some nodes connecting and some not to random other nodes. 
This is why we can hold connection info for any node and give it to any other. Later (post-launch we will enable hole punch and that will be an indirection, but complex for now)

Reply:
We should probably rename the field, then (JP mentioned this in another comment).